server/.env.example

```
@@ -1,30 +1,60 @@
+# Authentication (REQUIRED - generate strong secrets!)
+# Generate: openssl rand -base64 48
+JWT_SECRET=replace_with_generated_secret_min_32_chars
+REFRESH_TOKEN_SECRET=replace_with_different_generated_secret_min_32_chars
 
-# Client
-CLIENT_URL=http://localhost:5173
+# Email Configuration (optional for demo)
+# If email features aren't needed, leave as-is; 
+# the app will still work but email features will be disabled.
+# Redis
+REDIS_URL=redis://redis:6379
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The REDIS_URL key is duplicated

(DuplicatedKey)